react/FileSelector/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const FileSelector = ({
+  className,
+  disabled,
+  hidden,
+  multiple,
+  processSelected,
+  renderContent,
+  ...inputProps
+}) => (
+  <label role="button" disabled={disabled} className={className}>
```

Comment:
A label [can't have a `role`](https://www.w3.org/TR/html5/forms.html#the-label-element). It should be on the `span` or whatever child you choose.